filters/DividerFilter.cpp

```diff
@@ -85,27 +114,63 @@ void DividerFilter::addArgs(ProgramArgs& args)
     args.add("mode", "A mode of 'partition' will write sequential points "
         "to an output view until the view meets its predetermined size. "
         "'round_robin' mode will iterate through the output views as it "
-        "writes sequential points.", m_mode, DividerFilter::Mode::Partition);
-    m_cntArg = &args.add("count", "Number of output views", m_size);
-    m_capArg = &args.add("capacity", "Maximum number of points in each "
-        "output view", m_size);
+        "writes sequential points. A mode of 'split' will output new "
+        "views every time a 'capacity' number of points with the given 'expression'"
+        "are visited", m_args->m_mode, DividerFilter::Mode::Partition);
```

Comment:
Here I'm seeing `expression` combined with `capacity` and not `count` (which I think was the intent), but it's the first I've heard of a mode called `split`!

Reply:
after thinking about this some more, I dropped the `count` thing entirely. All this now does is split based on the expression passing. You can control it by stacking together other filters.